### DISCUSSION TITLE
How can I add "Startmenu2" in GoTo ?

### DISCUSSION THREAD

**Discussion by Eta-Beta** [moudey/Shell#279] [Q&A]
Hello, in goto there is the "Startmenu" element which opens the "C:\Users\Giorgio\AppData\Roaming\Microsoft\Windows\Start Menu" folder. I would like to add a second element "Startmenu2" which opens the folder "C:\ProgramData\Microsoft\Windows\Start Menu". I duplicated the item but I don't know the syntax to enter the address of the folder to open. Thank you.

**Comment by moudey** [comment]
```
item(title="Startmenu2" cmd='@sys.programdata\Microsoft\Windows\Start Menu')
```

**Reply by Eta-Beta** [reply]
Hi, it works and thanks as always, I added the icon 

`item(title="Startmenu2" image=inherit cmd='@sys.programdata\Microsoft\Windows\Start Menu')`

P.S. I forgot, if I want to start a program I simply add item(title="Startmenu2" image=inherit cmd='@sys.programdata\Microsoft\Windows\Start Menu\prova.exe') ?
While if you wanted to open a folder located on disk D for example what would change?
